Configure

```diff
@@ -1108,25 +1108,18 @@ if (!$disabled{dso} && $target{dso_scheme} ne "")
 $config{ex_libs}="$libs$config{ex_libs}" if ($libs ne "");
 
 # If threads aren't disabled, check how possible they are
-unless ($disabled{threads}) {
-    if ($auto_threads) {
-        # Enabled by default, disable it forcibly if unavailable
-        if ($target{thread_scheme} eq "(unknown)") {
-            $disabled{threads} = "unavailable";
-        }
-    } else {
-        # The user chose to enable threads explicitly, let's see
-        # if there's a chance that's possible
-        if ($target{thread_scheme} eq "(unknown)") {
-            # If the user asked for "threads" and we don't have internal
-            # knowledge how to do it, [s]he is expected to provide any
-            # system-dependent compiler options that are necessary.  We
-            # can't truly check that the given options are correct, but
-            # we expect the user to know what [s]He is doing.
-            if ($no_user_cflags && $no_user_defines) {
-                die "You asked for multi-threading support, but didn't\n"
-                    ,"provide any system-specific compiler options\n";
-            }
+unless ($disabled{threads} || $auto_threads) {
+    # The user chose to enable threads explicitly, let's see
+    # if there's a chance that's possible
+    if ($target{thread_scheme} eq "(unknown)") {
+        # If the user asked for "threads" and we don't have internal
+        # knowledge how to do it, [s]he is expected to provide any
+        # system-dependent compiler options that are necessary.  We
+        # can't truly check that the given options are correct, but
+        # we expect the user to know what [s]He is doing.
+        if ($no_user_cflags && $no_user_defines) {
```

Comment:
As already mentioned elsewhere (as a side note), these two are always 0 here, so this condition is never met.

Reply:
I missed that...  or have forgotten.  I'll try to remember for another commit.